Test/Unit/Magento/Framework/App/AreaListTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\UpwardConnector\Test\Unit\Magento\Framework\App;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\UpwardConnector\Plugin\Magento\Framework\App\AreaList as AreaListPlugin;
+
+class AreaListTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var ScopeConfigInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $scopeConfig;
+
+    /**
+     * \Magento\Framework\App\AreaList|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $areaListMock;
+
+    /**
+     * @var AreaListPlugin
+     */
+    private $areaListPlugin;
+
+    protected function setUp()
+    {
+        $this->scopeConfig = $this->createMock(ScopeConfigInterface::class);
+        $this->areaListMock = $this->createMock(\Magento\Framework\App\AreaList::class);
+
+        $this->scopeConfig->expects($this->any())->method('getValue')->willReturn(
+            "foo\r\nbar"
+        );
+
+        $objectManagerHelper = new \Magento\Framework\TestFramework\Unit\Helper\ObjectManager($this);
+        $this->areaListPlugin = $objectManagerHelper->getObject(AreaListPlugin::class, [
```

Comment:
Multiline function arguments should all be in new lines: https://github.com/magento/magento-coding-standard

Reply:
Fixed by d8bde4b.

I did run this code through `phpcs`, and none of these suggestions popped up. This is what I ran, should I be doing something different?

`vendor/bin/phpcs --standard=Magento2 ~/Projects/magento2-upward-connector`

Reply:
@tjwiebell huh weird, I'd add it to your IDE as well. Maybe CS doesn't contain these random rules from our Jenkins env.